src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -1151,7 +1152,7 @@ public static bool ContainsDynamic(this TypeSymbol type)
 
         internal static bool ContainsNativeInteger(this TypeSymbol type)
```

Comment:
>ContainsNativeInteger [](http://example.com/codeflow?start=29&length=21)

Perhaps helpers like this one should be renamed too in order to avoid confusion?